### ISSUE TITLE
error running demo 

### ISSUE THREAD

**Issue by trops** [anewell/pose-hg-demo#3]
HI there, I may have done something wrong when trying to install and run the code, but I think I followed all requirements, and am getting the following error when trying to execute the demo.

qlua main.lua demo

qlua: ...h/install/share/lua/5.1/nn/SpatialBatchNormalization.lua:83: attempt to index field 'running_std' (a nil value)

Am I missing a library? Thanks for the help!

**Comment by anewell** [comment]
I've seen that error with either an outdated version of Torch or the pretrained model (don't remember which), so one or the other likely needs an update. If that doesn't take care of your issue, let me know and I can look into it further.

**Comment by trops** [comment]
HI thanks for the reply, I upgraded Torch and still have the issue. You mentioned updating the model itself, Im new to this code and was wondering how I would update the model?

**Comment by anewell** [comment]
I just mentioned that in case you happened to have downloaded the model months ago, which I don't think is the issue. I still think the likely culprit is an outdated version of nn or cunn which you can update with 'luarocks install nn' and 'luarocks install cunn'. The batch normalization code in Torch hasn't used the field 'running_std' for a while. Also, see this issue: https://github.com/facebook/fb.resnet.torch/issues/3
